etc/adminhtml/system.xml

```diff
@@ -230,6 +230,15 @@
                     <label>Delivery enabled</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="delivery_fee" translate="label comment" type="text" sortOrder="305" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Delivery fee</label>
```

Comment:
Zou ‘Delivery fee’ nog vertaald kunnen / moeten worden voor de netheid? (Ik denk dat die andere teksten eronder wel al in de vertalingen staan?)

Reply:
Is nu gedaan